src/operations/aggregate.ts

```diff
@@ -7,13 +7,22 @@ import { Aspect, defineAspects } from './operation';
 const DB_AGGREGATE_COLLECTION = 1;
 const MIN_WIRE_VERSION_$OUT_READ_CONCERN_SUPPORT = 8;
 
-class AggregateOperation extends CommandOperation {
+interface AggregateOperationOptions extends CommandOperation {
+  batchSize?: any;
```

Comment:
Please add types for all of these options, and move documentation over from the `aggregate` call in `collection.ts` so that we have high quality documentation